src/SDKs/ServiceBus/Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.csproj

```diff
@@ -10,6 +10,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+    <Version>0.2.0-preview</Version>
```

Comment:
@v-Ajnava <versionPrefix> is good enough for the version info. Either use versionPrefix or version but not both.
Also remove all commented code from .csproj files